sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -192,10 +192,11 @@ struct _pi_device : _pi_object {
 };
 
 struct _pi_context : _pi_object {
-  _pi_context(pi_uint32 NumDevices, const pi_device *Devs)
-      : Devices{Devs, Devs + NumDevices}, ZeCommandListInit{nullptr},
-        ZeEventPool{nullptr}, NumEventsAvailableInEventPool{},
-        NumEventsLiveInEventPool{} {
+  _pi_context(pi_uint32 NumDevices, const pi_device *Devs,
+              ze_context_handle_t ZeContext)
```

Comment:
Please make the native L0 handle be the first argument (like in other constructors)

Reply:
Fixed.